Tasks/AzureTestPlanV0/task.json

```diff
@@ -77,13 +77,31 @@
       "label": "Select Test framework language",
       "helpMarkDown": "Test Framework Language of automated tests in test plan",
       "options": {
-        "Java-Maven": "Java-Maven",
-        "Java-Gradle": "Java-Gradle",
+        "JavaMaven": "Java-Maven",
+        "JavaGradle": "Java-Gradle",
         "Python": "Python",
         "Go": "Go",
         "Jest": "Jest"
       }
     },
+    {
+      "name": "pomFilePath",
+      "type": "string",
+      "label": "Pom file path",
+      "defaultValue": "",
+      "required": false,
+      "helpMarkDown": "Add custom file path of pom file incase it is not pom.xml at root directory",
```

Comment:
Let's have the description as "Relative path from the repository root to the Maven POM file." 

This ensure consistent experience across all the tasks. For ref, maven: [Link](https://github.com/microsoft/azure-pipelines-tasks/blob/18b2a6e45201b869e5439f15ea32129602b3babf/Tasks/MavenV4/task.json#L62C5-L72C7) and gradle: [Link](https://github.com/microsoft/azure-pipelines-tasks/blob/18b2a6e45201b869e5439f15ea32129602b3babf/Tasks/GradleV3/task.json#L48C5-L58C7)

Reply:
Updated help markdown text for maven